Sources/Common/RxManager.swift

```diff
@@ -33,7 +33,25 @@ extension RxManager {
             }
         }
     }
-    
+
+    /**
+     Returns an `Observable` of T for the current RouteType. Notice that T conforms to OperaDecodable. If something goes wrong a Opera.Error error is propagated through the result sequence.
+
+     - parameter keyPath: keyPath to look up json object to serialize. Ignore parameter or pass nil when json object is the json root item.
+
+     - returns: An instance of `Observable<T>` filled with sample data specified on the RouteType.
+     */
+    public func sampleObject<T: OperaDecodable>(_ route: RouteType, keyPath: String? = nil) -> Observable<T> {
```

Comment:
I agree, do you think it's also necessary to make `collection`, `object`and `any` methods protected? @fedeojeda95 @m-revetria